crates/bevy_ecs/src/world/mod.rs

```diff
@@ -2085,6 +2085,18 @@ impl World {
         self.storages.resources.clear();
         self.storages.non_send_resources.clear();
     }
+
+    /// Initializes all of the components in the given [`Bundle`] and returns their ids.
+    ///
+    /// This is equivalent to calling [`init_component`](Self::init_component) on each component
+    /// in the bundle.
+    #[inline]
+    pub fn init_bundle<B: Bundle>(&mut self) -> &[ComponentId] {
+        let id = self
+            .bundles
+            .init_info::<B>(&mut self.components, &mut self.storages);
+        self.bundles.get(id).unwrap().components()
```

Comment:
```suggestion
        // SAFETY: The bundle was just initialized. The bundle must still exist.
        unsafe { self.bundles.get(id).debug_unwrap_unchecked().components() }
```

We should be avoid panicking in the ECS when we know the invariants are satisfied. Ideally `init_info` should return a `&BundleInfo` to avoid the need for this, but that's out of the scope of the PR.

Reply:
I've changed it to `unwrap_unchecked`, I couldn't find a `debug_unwrap_unchecked` anywhere?

Reply:
DebugCheckedUnwrap is an extension trait we have inside bevy_ecs. It's identical but panics in debug mode if the Option isn't actually Some.